### PR TITLE
Nerfs Fuel Tanks

### PR DESCRIPTION
**Pull request by LadyFowls**
- tweak: "Changes how exploding fuel tanks work. Only leaking fuel tanks will blow up when shot."
  - tweak: "Changes the explosion severity of fuel tanks."
  - tweak: "Shooting a tank now has a chance to cause a bullet hole leak."

### CODE REVIEW

**Comment by LadyFowls** [comment]
!review

**Comment by alsoandanswer** [comment]
This needs feedback thread

Fuel tanks have been nerfed time and time again and they have like a 50/50 chance of being actually lethal at all at someone at point-blank range.

This is superfluous at best and potentially mechanic removal at worst

**Comment by BurgerLUA** [comment]
When have fuel tanks been nerfed in the last 6 months?

**Comment by alsoandanswer** [comment]
That's not the point here. Anyway, discussion like this should be kept to a feedback thread...which needs to be created.

File: code/modules/reagents/reagent_dispenser.dm [hunk 1]
```diff
@@ -196,32 +196,40 @@
...
+				is_leaking = TRUE
 
 /obj/structure/reagent_dispensers/fueltank/ex_act(var/severity = 3.0)
 
 	if (QDELETED(src))
 		return
 
 	if (reagents.total_volume > 500)
-		explosion(src.loc,1,2,4)
+		explosion(src.loc,-1,3,2)
```
Comment (Lohikar) [inline code/modules/reagents/reagent_dispenser.dm]: Today in explosioncode fun facts: this change does largely bugger-all if live uses recursive/iterative explosions. Which I'm 95% sure it does.

Comment (Lohikar) [reply code/modules/reagents/reagent_dispenser.dm]: Double-checking the code, this just reduces the effective power from 8 to 5.

**Comment by NonQueueingMatt** [comment]
Unaddressed comments, closing.

**Pull request closed without merging**